tidy3d/plugins/dispersion/fit_web.py

```diff
@@ -31,8 +31,16 @@ class AdvancedFitterParam(Tidy3dBaseModel):
     bound_f: NonNegativeFloat = Field(
         None,
         title="Upper bound of pole frequency",
-        description="Upper bound of real and imaginary part of pole "
-        "frequency ``a`` in the model :class:`.PoleResidue` (The default 'None' will trigger "
+        description="Upper bound of real and imaginary part of ``a`` that corresponds to pole "
+        "damping rate and frequency in the model :class:`.PoleResidue` (The default 'None' "
+        "will trigger automatic setup based on the frequency range of interest).",
+        units=HERTZ,
+    )
+    bound_lower_f: NonNegativeFloat = Field(
+        0.0,
+        title="Lower bound of pole frequency",
+        description="Lower bound of imaginary part of ``a`` that corresponds to pole "
+        "frequency in the model :class:`.PoleResidue` (The default 'None' will trigger "
```

Comment:
default is `0` so maybe we should change the description? also 'None' -> ``None`` for the description text.